File: examples/functions/next-style/admin/post.users.[id].py
import json


def handler(event):
    return {
        "status": 200,
        "headers": {"Content-Type": "application/json"},
        "body": json.dumps({
            "route": "POST /admin/users/:id",
            "params": event.get("params") or {},
            "runtime": "python",
        }),
    }

File: tests/fixtures/multi-root/nextstyle-clean/admin/post.users.[id].js
exports.handler = async (event) => {
  const id = (event.path_params || {}).id || null;
  return {
    status: 200,
    headers: { "Content-Type": "application/json" },
    body: JSON.stringify({
      runtime: "node",
      route: "POST /nextstyle-clean/admin/users/:id",
      id,
      event,
    }),
  };
};

